scripts/release.sh

```diff
@@ -224,7 +224,7 @@ fi
 echo $files_to_revert | xargs git checkout --
 
 $git add * .nojekyll
-$git commit --author="$GIT_AUTHOR" -m "Publish docs website $version_majorminor"
```

Comment:
taken from https://stackoverflow.com/questions/8123674/how-to-git-commit-nothing-without-an-error